test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpExporterTest.cs

```diff
@@ -268,6 +268,31 @@ public void ToOtlpSpanTest()
             }
         }
 
+        [Fact]
+        public void ToOtlpSpanNullArraysTest()
+        {
+            using var activitySource = new ActivitySource(nameof(this.ToOtlpSpanTest));
+
+            using var rootActivity = activitySource.StartActivity("root", ActivityKind.Client);
+
+            var stringArr = new string[] { "test", null };
+            var longArr = new long?[] { 1, null };
```

Comment:
what happens to this null element ?

Reply:
I removed the tests for nullable primtive type arrays like long?[] and double?[]. I think discussion of them is outside the scope of this PR